Integrations/GoogleBigQuery/GoogleBigQuery.yml

```diff
@@ -0,0 +1,96 @@
+  key
+configuration:
+- display: Google service account JSON (a credentials JSON generated from Google API
+    Manager or from GCP console)
+  name: google_service_creds
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: ''
+  type: python
+  commands:
+  - name: bigquery-query
+    arguments:
+    - name: query
+      required: true
+      description: A query string, following the BigQuery query syntax, of the query
+        to execute.
+    - name: location
+      description: The geographic location where the job should run. Required except
+        for US and EU.
+    - name: allow_large_results
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+      description: Allow large query results tables (legacy SQL, only)
+    - name: default_dataset
+      description: A stringof the fully-qualified dataset ID in standard SQL format.
+        The value must included a project ID and dataset ID separated by dots.
+    - name: destination_table
+      description: The table where results are written. Default value is None if not
+        set.
+    - name: kms_key_name
+      description: Custom encryption configuration for the destination table.
+    - name: dry_run
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+      description: If set to true, BigQuery doesn't run the job. Instead, if the query
+        is valid, BigQuery returns statistics about the job such as how many bytes
+        would be processed. If the query is invalid, an error returns. The default
+        value is false.
+    - name: priority
+      auto: PREDEFINED
+      predefined:
+      - BATCH
+      - INTERACTIVE
+      description: Priority of the query. Possible values include INTERACTIVE and
+        BATCH. The default value is INTERACTIVE.
+    - name: use_query_cache
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+      description: Look for the query result in the cache.
+    - name: use_legacy_sql
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+      description: Use legacy SQL syntax.
+    - name: job_id
+      description: The ID of the job. The ID must contain only letters (a-z, A-Z),
+        numbers (0-9), underscores (_), or dashes (-). The maximum length is 1,024
+        characters.
+    - name: write_disposition
+      auto: PREDEFINED
+      predefined:
+      - WRITE_EMPTY
+      - WRITE_TRUNCATE
+      - WRITE_APPEND
+      description: Specifies the action that occurs if destination table already exists.
+    outputs:
+    - contextPath: BigQuery.Query
+      description: The query performed.
+      type: String
+    - contextPath: BigQuery.Row
+      description: The table rows the given query returned.
+      type: Unknown
+    description: Perform a query on BigQuery
+  dockerimage: bigquery_image_local_3
+  runonce: false
+tests: Google BigQuery Test
```

Comment:
not needed

Reply:
Why not? And if not, why are tests ever specified that way?